spaceship/lib/spaceship/portal/app.rb

```diff
@@ -15,6 +27,10 @@ class App < PortalBase
       # @return (String) the supported platform of this app
       # @example
       #   "ios"
+      #   "mac"
```

Comment:
@bassrock I'm not sure this is correct. Where did you find this data? I've only seen "ios", "mac" and "safari" in the dev portal. iOS and tvOS are all bundled under the same endpoint. Also it looks there is no representation of an app in the Safari section of the dev portal.